<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_send">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>发送区</string>
   </property>
   <widget class="QPushButton" name="pBtn_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_send_clear">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_timedSending">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>定时发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_send_interval">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>30</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>间隔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send_time">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>60</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_send_ms">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>30</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_send_stop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>448</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_receive">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>470</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>接收区</string>
   </property>
   <widget class="QPushButton" name="pBtn_receive_clear">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_receive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>450</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_saveData">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>保存数据</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_network">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>网络设置</string>
   </property>
   <widget class="QLabel" name="label_destnIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>目标IP：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_destnIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_destnPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>目标端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_destnPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>25565</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_listenPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>监听端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_listenPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>25565</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_listenStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开始监听</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_networkConfig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>本机网络信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sendCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送计数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receiveCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>接收计数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sendCount">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_receiveCount">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_count_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>计数清零</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pBtn_send</tabstop>
  <tabstop>pBtn_receive_clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
